db/post_migrate/20190103065652_remove_uploaded_meta_id_from_category.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUploadedMetaIdFromCategory < ActiveRecord::Migration[5.2]
```

Comment:
You need to `require 'migration/column_dropper'` I think. We've had problems where the migration will fail in the past because `Migration::ColumnDropper` wasn't available.